Dockerfile

```diff
@@ -1,52 +1,39 @@
-## semgrep build
-
-FROM python:3.7.7-alpine3.11 as build-semgrep
```

Comment:
A lot of this is no longer necessary now that we're not burning the commit hash into the package build/install. I.e. the `.git` directory is no longer necessary, and we can greatly simplify the install process (a simple `pip install` should do the trick).